app/views/admin/configs/show.html.erb

```diff
@@ -991,8 +990,8 @@
         <table class="table">
           <thead>
             <tr>
-              <th>Variable</th>
-              <th>Value</th>
+              <th scope="col">Variable</th>
```

Comment:
TIL about the [scope](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th#attr-scope) attribute.

Reply:
(me too :D)